quantum/quantum_keycodes.h

```diff
@@ -409,6 +409,11 @@ enum quantum_keycodes {
     KC_LAPO,  // 5CF6
     KC_RAPC,  // 5CF7
 
+    // Key Overrides
+    KEY_OVERRIDE_TOGGLE,
+    KEY_OVERRIDE_ON,
+    KEY_OVERRIDE_OFF,
+    
```

Comment:
Worth noting everything including and following the Sequencer keycodes will not be able to be supported by VIA, because the Sequencer keycodes do not have fixed values. See #11156

Reply:
Rebased and moved the keycodes